### PR TITLE
bugfix/ATC-64

### LINKED ISSUE #64: Readback given after A/C are cleared for T/O is buggy
https://github.com/zlsa/atc/issues/711

### PR DESCRIPTION
**Pull request by n8rzz**
fixes #64 (ref original issue https://github.com/zlsa/atc/issues/711)

Reverts !== to != for a very specific case and adds missing continue.

### CODE REVIEW

File: src/assets/scripts/aircraft/AircraftInstanceModel.js [hunk 1]
```diff
@@ -650,12 +650,13 @@ export default class Aircraft {
...
         const deferred = [];
 
-        for (let i = 0; i < commands.length; i += 1) {
+        for (let i = 0; i < commands.length; i++) {
             const command = commands[i][0];
             const args = commands[i].splice(1);
 
             if (command === FLIGHT_MODES.TAKEOFF) {
                 deferred.push([command, args]);
+                continue
```
Comment (erikquinn) [inline src/assets/scripts/aircraft/AircraftInstanceModel.js]: `continue;`

**Review by erikquinn — COMMENTED** [review]
It technically would work as-is, but I assume you would want to amend this.

File: src/assets/scripts/aircraft/AircraftInstanceModel.js (hunk 1 above)
Comment (n8rzz) [reply src/assets/scripts/aircraft/AircraftInstanceModel.js]: Good catch!  That is exactly why I had you review these, there's usually something small that gets missed that another set of eyes will see.

You are correct, babel takes care of that at transpile time, but I'd like for that semi colon to be there.

Comment (n8rzz) [reply src/assets/scripts/aircraft/AircraftInstanceModel.js]: fixed

**Review by erikquinn — APPROVED** [review]

**Pull request merged**